src/Components/App.jsx

```diff
@@ -12,25 +13,27 @@ class App extends React.Component {
 
   componentDidMount() {
     const setState = (value) => {
-      this.setState({data: value});
+      this.setState({data: value });
     }
     console.log('Component Mounted');
     $.ajax({
       method: 'GET',
       url: 'http://localhost:3000/relatedlisting',
       contentType: 'application/json',
       success: (data) => {
-        setState(data);
+        setState(JSON.parse(data));
       },
       error: (error) => {
-        console.log(error)
+        console.log(error);

```

Comment:
Consider a different error handler that will show up on the client. But its up to you!